channels/uk.m3u

```diff
@@ -215,3 +215,69 @@ https://de1se01.v2beat.live/playlist.m3u8
 https://wonder-samsung-uk.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="Dorama.uk" tvg-name="Дорама" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Movies",Дорама
 http://188.40.68.167/russia/dorama/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/EuroNews_400x400.png?raw=true" group-title="",Euronews
+https://d1mpprlbe8tn2j.cloudfront.net/v1/master/7b67fbda7ab859400a821e9aa0deda20ab7ca3d2/euronewsLive/87O7AhxRUdeeIVqf/ewnsabren_eng.m3u8
+#EXTINF:-1 tvg-id="886ad6" tvg-name="Euronews" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/8ElFwvm.png" group-title="",Euronews
+https://euronews-euronews-world-1.plex.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Euronews
+https://euronews-euronews-world-1.roku.wurl.com/manifest/b34060fcaf527710b244a3e16504edd0.m3u8
+#EXTINF:-1 tvg-id="8268c2" tvg-name="Euronews_Español" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/8ElFwvm.png" group-title="",Euronews Español
+https://euronews-euronews-spanish-2.plex.wurl.com/manifest/playlist.m3u8
```

Comment:
As I know Euronews is a French channel: https://www.lyngsat.com/tvchannels/fr/EuroNews-English.html